doc/diag_comp_overall_architecture.uxf

```diff
@@ -42,347 +67,405 @@ UDP package</panel_attributes>
-      <x>371</x>
-      <y>392</y>
-      <w>70</w>
-      <h>56</h>
+      <x>552</x>
+      <y>534</y>
+      <w>120</w>
+      <h>48</h>
     </coordinates>
     <panel_attributes>lt=-&gt;
-StartReply
-received</panel_attributes>
+RawReplyReceived [isStartReply()]
+group=1</panel_attributes>
     <additional_attributes>10.0;10.0;10.0;60.0</additional_attributes>
   </element>
   <element>
     <id>Relation</id>
     <coordinates>
-      <x>371</x>
-      <y>476</y>
-      <w>56</w>
-      <h>56</h>
+      <x>552</x>
+      <y>588</y>
+      <w>54</w>
+      <h>66</h>
     </coordinates>
     <panel_attributes>lt=-&gt;
-Stop 
-request</panel_attributes>
-    <additional_attributes>10.0;10.0;10.0;60.0</additional_attributes>
+StopRequest
+group=1</panel_attributes>
+    <additional_attributes>10.0;10.0;10.0;90.0</additional_attributes>
   </element>
   <element>
     <id>UMLState</id>
     <coordinates>
-      <x>343</x>
-      <y>518</y>
-      <w>77</w>
-      <h>28</h>
+      <x>504</x>
+      <y>642</y>
+      <w>108</w>
+      <h>24</h>
     </coordinates>
-    <panel_attributes>Wait for
-stop reply</panel_attributes>
+    <panel_attributes>halign=center
+valign=center
+WaitForStopReply
+group=1</panel_attributes>
     <additional_attributes/>
   </element>
   <element>
     <id>Relation</id>
     <coordinates>
-      <x>259</x>
-      <y>315</y>
-      <w>224</w>
-      <h>259</h>
-    </coordinates>
-    <panel_attributes>lt=-&gt;
-m1=StopReply received</panel_attributes>
-    <additional_attributes>170.0;330.0;170.0;350.0;10.0;350.0;10.0;10.0;120.0;10.0</additional_attributes>
-  </element>
-  <element>
-    <id>Relation</id>
-    <coordinates>
-      <x>413</x>
-      <y>434</y>
-      <w>98</w>
-      <h>49</h>
+      <x>606</x>
+      <y>570</y>
+      <w>120</w>
+      <h>30</h>
     </coordinates>
     <panel_attributes>lt=-&gt;
-Monitoring 
-frame
-received</panel_attributes>
-    <additional_attributes>10.0;10.0;50.0;10.0;50.0;50.0;10.0;50.0</additional_attributes>
+RawMonitoringFrameReceived
+group=1</panel_attributes>
+    <additional_attributes>10.0;10.0;30.0;10.0;30.0;30.0;10.0;30.0</additional_attributes>
   </element>
   <element>
     <id>UMLGeneric</id>
     <coordinates>
-      <x>224</x>
-      <y>252</y>
-      <w>287</w>
-      <h>329</h>
+      <x>384</x>
+      <y>384</y>
+      <w>378</w>
+      <h>348</h>
     </coordinates>
     <panel_attributes>symbol=component
+style=wordwrap
+layer=0
+group=1
 &lt;&lt;State machine&gt;&gt;
-Scanner Controller
-(possible realization; not fixed)
---</panel_attributes>
-    <additional_attributes/>
-  </element>
-  <element>
-    <id>Relation</id>
-    <coordinates>
-      <x>504</x>
-      <y>406</y>
-      <w>49</w>
-      <h>28</h>
-    </coordinates>
-    <panel_attributes>lt=..
+ScannerProtocolDef
+--
+**Explanation:**
+This component represents the scanner protocol/ state machine (indirectly) described in the Communication manual of the scanner on page 1.
```

Comment:
```suggestion
This component represents the scanner protocol / state machine (indirectly) described in the communication manual of the scanner on page 1.
```

Reply:
And: where can I find this manual? Is there a link somewhere? Does the user even have access to this?